.github/workflows/continuous-integration.yml

```diff
@@ -487,7 +487,7 @@ jobs:
           restore-keys: "php-${{ matrix.php-version }}-composer-${{ matrix.dependencies }}-"
 
       - name: "Require Roave/BackwardCompatibilityCheck"
-        run: "composer require --no-update --no-interaction --prefer-dist --prefer-stable --dev \"roave/backward-compatibility-check:6.0.x-dev\""
+        run: "composer require --no-update --no-interaction --prefer-dist --prefer-stable --dev \"roave/backward-compatibility-check:6.0.x-dev\" \"roave/better-reflection:5.0.x-dev\""
```

Comment:
Do you know when the stable release for these tools is planned?

Reply:
We're trying to get it done ASAP; depending on it in "dev" state is not a big deal, it's just it's a moving target, plus there's a bug I discovered in Roave/BackwardCompatibilityCheck#360

Reply:
@mimmi20 I'll get this fixed upstream and manage the merge of this from now on